File: app/src/main/java/ru/github/pvtitov/senatapp/pojos/CurrentVersion.java
package ru.github.pvtitov.senatapp.pojos;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CurrentVersion {

    @SerializedName("version")
    @Expose
    private int version;
    @SerializedName("fileName")
    @Expose
    private String fileName;
    @SerializedName("createdBy")
    @Expose
    private CreatedBy createdBy;
    @SerializedName("createdAt")
    @Expose
    private String createdAt;
    @SerializedName("size")
    @Expose
    private int size;
    @SerializedName("encrypted")
    @Expose
    private boolean encrypted;

    public int getVersion() {
        return version;
    }

    public void setVersion(int version) {
        this.version = version;
    }

    public String getFileName() {
        return fileName;
    }

    public void setFileName(String fileName) {
        this.fileName = fileName;
    }

    public CreatedBy getCreatedBy() {
        return createdBy;
    }

    public void setCreatedBy(CreatedBy createdBy) {
        this.createdBy = createdBy;
    }

    public String getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public int getSize() {
        return size;
    }

    public void setSize(int size) {
        this.size = size;
    }

    public boolean isEncrypted() {
        return encrypted;
    }

    public void setEncrypted(boolean encrypted) {
        this.encrypted = encrypted;
    }

}

File: app/src/main/java/ru/github/pvtitov/senatapp/pojos/CurrentVersion.kt
package ru.github.pvtitov.senatapp.pojos

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CurrentVersion {

    @SerializedName("version")
    @Expose
    var version: Int = 0
    @SerializedName("fileName")
    @Expose
    var fileName: String = ""
    @SerializedName("createdBy")
    @Expose
    var createdBy: CreatedBy? = null
    @SerializedName("createdAt")
    @Expose
    var createdAt: String = ""
    @SerializedName("size")
    @Expose
    var size: Int = 0
    @SerializedName("encrypted")
    @Expose
    var isEncrypted: Boolean = false

}
